templates/placementapi/config/placement-api-config.json

```diff
@@ -24,6 +24,27 @@
             "dest": "/etc/placement/logging.conf",
             "owner": "root",
             "perm": "0644"
+        },
+        {
+            "source": "/var/lib/config-data/tls-certificates/tls.key",
+            "dest": "/etc/pki/tls/private/tls.key",
+            "owner": "placement",
+            "perm": "0600",
+            "optional": true
+        },
+        {
+            "source": "/var/lib/config-data/tls-certificates/tls.crt",
+            "dest": "/etc/pki/tls/certs/tls.crt",
+            "owner": "placement",
+            "perm": "0755",
+            "optional": true
+        },
+        {
+            "source": "/var/lib/config-data/ca-trust",
```

Comment:
@stuggi 
Since we will be using one CA bundle file created by openstack-operator, we can specify the individual file instead  the general folder (?)

Reply:
yes lets do that